packages/trader/src/Modules/SmartChart/Components/Markers/accumulators-profit-loss-text.tsx

```diff
@@ -1,10 +1,21 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Text } from '@deriv/components';
 import { FastMarker } from 'Modules/SmartChart';
 import classNames from 'classnames';
+import AccumulatorsProfitLossTooltip, { TRef } from './accumulators-profit-loss-tooltip';
 
-const ACTIONS = {
+type TAccumulatorsProfitLossText = Omit<
+    React.ComponentProps<typeof AccumulatorsProfitLossTooltip>,
+    'alignment' | 'exit_tick' | 'exit_tick_time' | 'high_barrier' | 'is_sold'
+>;
+
+type TActions = {
+    INC: string;
+    DEC: string;
+    ADD10: string;
+};
+
+const ACTIONS: TActions = {
```

Comment:
i thought we are trying to avoid typecasting with `as something`

Reply:
according to @farzin-deriv it's better to have `as const` for constants which enumerate something. `as const` makes the object type read-only explicitly and automatically fills its type with specific values instead of any string values. But it's optional. You can also leave it without `as const` and not add any type. In this case, TS will still be able to infer (figure out) the type of the object properties, it will know that they can be any strings.🙂 This is up to us if we want to specify whether we want to limit values to any strings or only specific values like 'increment' and etc.

Reply:
Nice 🚀 

@henry-deriv It's a bit confusing since they both use the `as` keyword 😅 

- `foo as something` is typecasting, We tell TS to consider `foo` to be `something`, So the TS keyword here is just `as` itself.
- `foo as const` is not typecasting, We tell TS to consider `foo` as read-only so it can know it will never change so it will infer the types strictly, So the TS keyword here is `as const` which is different than `as`.

Let me know if you need more explanation 🙇🏻